### ISSUE TITLE
[Bug]Sometimes the channel drop down menu will fail.

### ISSUE THREAD

**Issue by Se7enMuting** [Alumian/CEPGP-Retail#149]
using 1.12.7
**The Error**
```
2x CEPGP\Button.lua:434: bad argument #1 to 'lower' (string expected, got nil)
[C]: in function `lower'
CEPGP\Button.lua:434: in function `initFunction'
CEPGP\Utility.lua:1200: in function `CEPGP_UIDropDownMenu_Initialize'
[string "*:OnShow"]:1: in function <[string "*:OnShow"]:1>
[C]: in function `Show'
[string "*:OnClick"]:3: in function <[string "*:OnClick"]:1>
Locals:
(*temporary) = nil
(*temporary) = "string expected, got nil"
```
**Describe the bug**
When I enter the dungeons, I can't switch the channel and lootChannel.

**To Reproduce**
Let the ID of trading channel or guild recruitment channel be greater than 4.
Enter into a dungeon.

**Cause analysis**
When you enter the dungeon, some channels will become nil automatically.(trading and guild recruitment )
If the ID of these channels is greater than 4, this bug will be caused. Even if no error is reported, the channels after these nil channels will not appear in the channel drop down menu.

**My Fix**
[commit](https://github.com/Se7enMuting/CEPGP-Retail/commit/82e257c4028e6d4d4c68fd4d9e2d46d7a22ec691)

**Comment by Alumian** [comment]
Thanks for submitting this bug report Se7enMuting. Due to a change that blizzard has made in 1.13.3, addons are no longer able to report to custom player-made channels. Because of this, I have withdrawn this functionality from CEPGP. This change has doubled up as a resolution for this issue. Please expect this change in the 1.12.8 release.

**Comment by Alumian** [comment]
Resolved in the 1.12.8 release.
